crates/ruff/src/rules/flake8_bugbear/snapshots/ruff__rules__flake8_bugbear__tests__B008_B006_B008.py.snap

```diff
@@ -46,37 +46,6 @@ B006_B008.py:119:30: B008 Do not perform function call in argument defaults
 120 |     ...
     |
 
-B006_B008.py:157:34: B008 Do not perform function call `float` in argument defaults
-    |
-157 | def float_infinity_literal(value=float("1e999")):
-    |                                  ^^^^^^^^^^^^^^ B008
-158 |     pass
-    |
-
-B006_B008.py:162:30: B008 Do not perform function call `float` in argument defaults
```

Comment:
We should probably update these comments, they're not out-of-date, I think? (Can we also remove the special-casing code in the bugbear module, which should now be redundant.)

Reply:
I'm not sure I follow exactly on the comments. We settled on allowing `float` calls now right? So these snapshots can go. 

I updated the comments in the `.py` fixture to reflect this. As for the Violating string it's formatted like so:

```rust
format!("Do not perform function call `{name}` in argument defaults")
```

So this needs no edit. Does that cover what you mean or are you referring to something else. 

As for the special casing float logic, I'm removing it now!

Reply:
Sorry, I meant "they're out-of-date", not "they're _not_ out-of-date". What you've done here looks correct to me.

Reply:
Ah check! Removed infinity/nan logic now as well!